ogip/spec.py

```diff
@@ -83,12 +83,37 @@ def from_file_name_osa(fn):
 
     @staticmethod
     def from_file_name_normal(fn):
+        
         f = fits.open(fn)
+        if 'RATE' in f['SPECTRUM'].data.names:
```

Comment:
Could you please put this change in a different PR so it can be assessed separately?

Reply:
Done